scripts/kube/deploy.sh

```diff
@@ -146,9 +146,11 @@ fi
 kubectl delete -f $DIRNAME/postgres.yaml
 kubectl delete -f $BUILD_DIRNAME/airflow.yaml
 kubectl delete -f $DIRNAME/secrets.yaml
+kubectl delete -f $DIRNAME/namespace.yaml
```

Comment:
It's not advisable to delete namespace everytime we deploy. Some of the resources like PVC doesn't have to be removed inorder to redeploy an app

Reply:
Sure. It should be done now ^^